js/dialogcards.js

```diff
@@ -324,20 +321,25 @@ H5P.Dialogcards = (function ($, Audio, JoubelUI) {
   /**
    * Create card footer
    *
+   * @param {Object} card Card parameters
    * @returns {*|jQuery|HTMLElement} Card footer element
    */
-  C.prototype.createCardFooter = function () {
+  C.prototype.createCardFooter = function (card) {
     var self = this;
     var $cardFooter = $('<div>', {
       'class': 'h5p-dialogcards-card-footer'
     });
 
-    JoubelUI.createButton({
-      'class': 'h5p-dialogcards-turn',
-      'html': self.params.answer
-    }).click(function () {
-      self.turnCard($(this).parents('.h5p-dialogcards-cardwrap'));
-    }).appendTo($cardFooter);
+    //Create Turn button only if either answer or back image exists
+    if(card.answer != "" || card.backImage != undefined)
```

Comment:
card is undefined for self.createCardFooter() in line L#315, which makes this crash